GameServerApp/Program.cs

```diff
@@ -84,17 +139,17 @@ private static void Main(string[] args)
 
     public class ArgsOptions
     {
-        [Option("config", DefaultValue = "Settings/GameInfo.json")]
-        public string ConfigPath { get; set; }
+        [Option("config", DefaultValue = "Settings\\GameInfo.json")]
+        public string GameInfoJsonPath { get; set; }
 
-        [Option("config-gameserver", DefaultValue = "Settings/GameServerSettings.json")]
-        public string ConfigGameServerPath { get; set; }
+        [Option("config-gameserver", DefaultValue = "Settings\\GameServerSettings.json")]
```

Comment:
Use linux paths (unless they don't work on Windows, I believe they do in this case)